core/src/main/java/com/redhat/lightblue/client/LightblueClientConfiguration.java

```diff
@@ -32,6 +32,7 @@ public static Compression parseCompression(String str) {
     private ReadPreference readPreference;
     private String writeConcern;
     private Integer maxQueryTimeMS;
+    private transient Execution execution;
```

Comment:
Why is it different in this regard than other LightblueClientConfiguration fields?